bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableParser.scala

```diff
@@ -58,8 +58,14 @@ sealed trait MagnitudesFilter {
   def ignoredMagnitudeField(v: FieldId): Boolean = false
   // Indicates if a magnitude is valid
   def validMagnitude(m: Magnitude): Boolean = !(m.value.isNaN || m.error.exists(_.isNaN))
-  // Find what band the field descriptor should represent
-  def findBand(id: FieldId, band: String): Option[MagnitudeBand] = MagnitudeBand.all.find(_.name == band)
+
+  // Find what band the field descriptor should represent, in general prefer "upper case" bands over "lower case" Sloan bands.
+  // This will prefer U, R and I over u', r' and i' but will map "g" and "z" to the Sloan bands g' and z'.
+  def findBand(id: FieldId, band: String): Option[MagnitudeBand] =
```

Comment:
Right, so I think the question is if we want to leave this behavior as is right now (which matches the current test cases):

* Default/PPMXL: upper case band name first, then lower case (in order to match g' and z')
* UCAC4: g, r, i map to g', r', i' then follow rules for Default/PPMXL

or if we need to change that in any way? I personally think it's a bit weird that for the default case umag, gmag, rmag, imag and zmag result in standard bands (U, R, I) and Sloan bands (z', g' because there is no G and Z band) but then I am not sure if the test data in votable.xml is actually valid, because it's not PPMXL or UCAC4?

I guess either way, we can probably leave it as is and then refine those rules once we need to support other catalogs.